core/tee/tee_svc_storage.c

```diff
@@ -307,6 +307,12 @@ static TEE_Result tee_svc_storage_read_head(struct tee_ta_session *sess,
 
 	fops = o->pobj->fops;
 
+	o->pobj->head = malloc(sizeof(struct tee_svc_storage_head));
```

Comment:
But I still find a problem that, if A open the obj, and then B open the obj and write, after that A write, the condition seems that A will update the head without B's update?